tests/unit/rbtree/main.c

```diff
@@ -12,15 +12,23 @@
 	zassert_true(!!(n), "Tree check failed: [ " #n " ] @%d", __LINE__)
 
 #define MAX_NODES 256
+#define SCOPE 5
```

Comment:
I don't see "SCOPE" used here

Reply:
Thanks for review, and "SCOPE" has been removed.